Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py

```diff
@@ -130,6 +130,7 @@ def create_context(indicators, include_dbot_score=False):
                     'Type': indicator_type,
                     'Vendor': 'ThreatConnect'
                 }
+            indicators_dbot_score[value]['Reliability'] = params.get('integrationReliability')
```

Comment:
it is not a good practice to call params.get() within a func, please initilalize at `GLOBAL VARS` and call as global.